moon/bulk_test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# NASA qualification round number
+ROUND=3
+
+# after how long should the simulation give up (in seconds); note the round is allowed to run for up to 45 minutes
+TIMEOUT=1800
+
+# desired score
+SCORE="13"
+
+RANGE_START=0
+RANGE_END=49
+
+while getopts "r:t:s:f:l:h" opt; do
+    case ${opt} in
+        h)
+            echo "Usage:"
+            echo " -r <qualification round>"
+            echo " -t <timeout in secs>"
+            echo " -s <desired score>"
+            echo " -f <first seed range>"
+            echo " -l <last seed range>"
+            exit 0
+            ;;
+        r)
+            ROUND=$OPTARG
+            ;;
+        t)
+            TIMEOUT=$OPTARG
+            ;;
+        s)
+            SCORE=$OPTARG
+            ;;
+        f)
+            RANGE_START=$OPTARG
+            ;;
+        l)
+            RANGE_END=$OPTARG
+            ;;
+    esac
+done
+
+
+for i in $(seq $RANGE_START $RANGE_END)
+do
+
+    echo "==========" `date` "====== seed:" $i
+    $HOME/space-challenge/srcp2-competitors/docker/scripts/launch/roslaunch_docker -q -n --run-round $ROUND -s $i >& /dev/null
```

Comment:
we should unify the location of the scripts ... my are slightly different ... (under $HOME/git/) and it is not obvious, that it is not doing anything now

Reply:
Ideally there would be some universal method for finding the folder, maybe look for a folder named 'srcp2-competitors' in this user's home tree first.